MERN/Topic/Frontend/React/5.1_Connecting_with_API.md

```diff
@@ -0,0 +1,74 @@
+# **Getting started with API**
+
+## **How to connect with an API :**
+
+   <!-- <p> -->
+
+API stands for Application Program Interface, which can be defined as a set of methods of communication between various software components. In other words, an API allows software to communicate with another software.
+We can communicate and connect our frontend code with backend database using api.
```

Comment:
API is kind of confusing especially for beginners, Please provide an analogy for the same, which would help beginners to visualize it better. 
Example:You can give example of waiter, How it brings the food from backend(Kitchen) and serve it on frontend(Client-table).

Reply:
done !